src/sc_io.c

```diff
@@ -647,6 +647,12 @@ sc_io_nonuncompress (char *dest, size_t dest_size,
 
 #define SC_IO_ENCODE_INFO_LEN 9
 
+/* when SC_HAVE_ZLIB is undefined, we need to define Z_BEST_COMPRESSION (zlib.h compatibility) */
+#ifndef Z_BEST_COMPRESSION
+#define Z_BEST_COMPRESSION 9
```

Comment:
Thanks! What would you think about moving these three lines to the very top into the `#else` branch of
`#ifdef SC_HAVE_ZLIB`? Then we would survive if Z_BEST_COMPRESSION becomes a variable in zlib.

Reply:
agreed, it's better.